bluemira/geometry/wire.py

```diff
@@ -220,3 +220,17 @@ def change_placement(self, placement):
                 change_placement(o, placement._shape)
             else:
                 o.change_placement(placement)
+
+    def start_point(self) -> np.ndarray:
+        """
+        Get the coordinates of the start of the wire.
+        """
+        point = self.boundary[0].Vertexes[0].Point
```

Comment:
I'll have a think about this. We're trying to keep all the FreeCAD bits within the FreeCAD API (Vertexes and Point are FreeCAD objects, so in principle may not exist if we were to use some other CAD backend).